superset/proxy/api.py

```diff
@@ -158,14 +158,24 @@ def get_userid(self, user_id: str, **_kwargs: Any) -> Response:
         current logged in user, and will then send a request to Alfred to see
         if the passed in user_id is in any reports/incidents
 
-        :param user_id: String value representing the user id passed in from the front-end
+        :param user_id: String value representing the user id(s) passed in from the front-end
         :param _kwargs: Array representing any other arguments passed to the function
         :returns: Response generated from passing values to the make_alfred_connection function
         """
+        user_emails = user_id.split(",")
+        user_email_string = ''
+
+        if len(user_emails) > 1:
```

Comment:
I think we can check if len(user_emails) > 0 here and cut the elif. The for-loop shouldn't run if len(user_emails) == 1.

Reply:
Good catch, completely forgot that the for loop won't run if len(user_emails) == 1